napari/_qt/widgets/qt_welcome.py

```diff
@@ -67,12 +67,19 @@ def __init__(self, parent) -> None:
         )
         shortcut_layout.addRow(
             QtShortcutLabel(sc),
-            QtShortcutLabel(trans._('open image(s)')),
+            QtShortcutLabel(trans._('Open image(s)')),
+        )
+        sc = QKeySequence('Ctrl+Shift+P', QKeySequence.PortableText).toString(
```

Comment:
Next hardcoded shortcut. 
We need to remember this for future fix

Reply:
I made a tracking issue for command palette issues which includes that.
https://github.com/napari/napari/issues/7611